README.md

```diff
@@ -322,9 +322,10 @@ See the [developer documentation](https://github.com/Microsoft/vscode-mssql/wiki
 
 ## Repository Layout
 
-- `mssql/` – Primary MSSQL extension that provides connection management, editors, Copilot integration, notebooks, dashboards, and packaging tooling.
-- `sql-database-projects/` – SQL Database Projects extension focused on SQL project authoring, build, publish, and schema comparison experiences.
-- `typings/` – Shared `.d.ts` shims for first-party dependencies (azdata, dataworkspace, mssql, vscode-mssql).
+- `extensions/` - all of the individual VS Code extensions
```

Comment:
I wonder if the dev instructions are better placed in the wiki?  I'm not opposed to it here, though I tend to think of our readme as more a user-focused landing page, and less dev-focused.